spec/factories/collection_template.rb

```diff
@@ -26,10 +26,10 @@
 
     draft {
       collection_one.merge(
-        'TemplateName' => template_name || "#{Faker::Number.number(6)}_#{Faker::Superhero.name}",
-        'ShortName'    => draft_short_name || "#{Faker::Number.number(6)}_#{Faker::Superhero.name}",
+        'TemplateName' => template_name || "#{Faker::Number.number(digits: 6)}_#{Faker::Superhero.name}",
```

Comment:
I'm glad you took care of this. I wanted to make this change in my other PR but I felt it would have blown it up, in the sense that there would be so many extraneous files to look at.